src/components/DevicesView.jsx

```diff
@@ -96,11 +104,15 @@ class DevicesView extends Component {
                     <TableCell
                       className={classNames(
                         tableStyles['set-table-name'],
-                        tableStyles['coz-table-primary'],
-                        getDeviceKindClass(device.client_kind)
+                        tableStyles['coz-table-primary']
                       )}
                     >
-                      {device.client_name}
+                      <Media>
+                        <Img>
+                          <Icon icon={getDeviceIcon(device)} size={32} />
+                        </Img>
+                        <Bd className="u-ml-1">{device.client_name}</Bd>
+                      </Media>
```

Comment:
Should we use a CompositeRow instead?

Reply:
This is really a table row. The use of Media here is to vertically align things without adding custom classes.